web/src/test/java/com/bobocode/bring/web/RestControllerTest.java

```diff
@@ -26,21 +28,19 @@ public class RestControllerTest {
     public static final String NUMBER_PATH = "/example/number";
     public static final String REQUEST_PATH = "/example/request";
     public static final String RESPONSE_PATH = "/example/response";
-    public static final String BODY_AS_STRING = "/example/bodyAsString";
-    public static final String BODY_AS_ENTITY = "/example/bodyAsEntity";
     public static final String NOT_EXIST_PATH = "/not-exist";
-    public static final String HEADER = "/example/header";
     public static final String STATUS = "/example/status";
     public static final String CUSTOM_EXCEPTION_PATH = "/example/custom-exception";
     public static final String DEFAULT_EXCEPTION_PATH = "/example/default-exception";
     public static final String URL = "http://localhost:%s%s";
+    public static final String APPLICATION_JSON = "application/json";
     private static ObjectMapper objectMapper;
     private static ServerProperties serverProperties;
     private HttpClient httpClient;
 
     @BeforeAll
     static void beforeAll() {
-        BringApplicationContext bringApplicationContext = BringWebApplication.run("testdata");
+        BringApplicationContext bringApplicationContext = BringWebApplication.run("testdata.generalIntegrationTest");
```

Comment:
should be lowercase testdata.generallntegrationtest

Reply:
Done